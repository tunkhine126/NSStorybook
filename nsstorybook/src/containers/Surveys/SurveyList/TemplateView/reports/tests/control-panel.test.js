/**
 *
 * Tests for Control Panel
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import TestingComponent from 'components/shared/Testing';
import { render } from '@testing-library/react';

import ControlPanel from '../control-panel';

describe('<ControlPanel />', () => {
  const props = {
    onChange: () => {},
  };

  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(
      <TestingComponent>
        <ControlPanel {...props} />
      </TestingComponent>
    );
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <TestingComponent>
        <ControlPanel {...props} />
      </TestingComponent>
    );
    expect(firstChild).toMatchSnapshot();
  });
});
