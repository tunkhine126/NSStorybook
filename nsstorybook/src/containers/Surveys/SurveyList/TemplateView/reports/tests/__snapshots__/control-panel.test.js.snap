// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ControlPanel /> Should render and match the snapshot 1`] = `null`;
